functions/ts/kubeval/build/kubeval.Dockerfile

```diff
@@ -1,6 +1,15 @@
+FROM python:3.9-alpine3.12 as parser
+
+RUN pip install --no-cache-dir openapi2jsonschema
+
+COPY openapi.json /home/
+COPY scripts/preprocess.py /home/
+
+RUN python /home/preprocess.py /home/openapi.json | openapi2jsonschema --kubernetes --expanded --stand-alone -o /home/master-standalone file:///dev/stdin
```

Comment:
`preprocess.py` is a workaround for https://github.com/instrumenta/openapi2jsonschema/issues/49.
This step is really time-consuming. It takes 50+ seconds on my mac.

After we use openapi doc injected by kpt, we will need to do this conversion at runtime. I have some concerns about the UX, since it may be executing for 50s without any output.

Reply:
Maybe you can try other tools. There are some listed in https://openapi.tools/

Reply:
instrumenta/openapi2jsonschema is purposely built for kubeval and handle some special cases in k8s. I have spent some time trying to use some of tools in the list. They are need non-trivial amount of work to work with kubeval and k8s.
I'm going to fork `instrumenta/openapi2jsonschema` and fix the issue (instrumenta/openapi2jsonschema#49). And also improve it to write only the files that are needed which can reduce the time for conversion.